src/components/Card/Card.css

```diff
@@ -4,10 +4,12 @@
     flex-basis: 23%;
     list-style: none;
     margin-bottom: 5px;
+    cursor: pointer;
+    text-decoration: none;
 }
 
 .card-image {
-    width: 100%;
+    width: 296px;
```

Comment:
it's better to use max-widh if it's possible

Reply:
Added max-width